core/jazz_codeq/config/test-config.json

```diff
@@ -1,5 +1,5 @@
 {
-	"SONAR_USER": "{sonar_user}",
+	"SONAR_USER": "sonar_user",
```

Comment:
why was this removed?

Reply:
just for testing I had changed. Updated.